src/App.css

```diff
@@ -1,4 +1,283 @@
-.author-wrapper {
-  background-color: white;
-  margin-left: 2%;
+:root {
+  --border-color: #f1f1f1;
+  --backround-color: #fafafa;
+  --primary-color: #0290ff;
+  --accent-color: #8a8a8a;
+  --max-width: 1000px;
+}
+
+.nav-container {
+  display: grid;
+  place-items: center;
+  border-bottom: 2px solid var(--border-color);
+  margin-bottom: 2%;
+  margin-left: -5rem;
+}
+
+.nav-wrapper {
+  max-width: var(--max-width);
+  width: 80%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.left-wrapper {
+  display: flex;
+  align-items: center;
+  gap: 30px;
+}
+
+.nav-wrapper .nav-title {
+  color: var(--primary-color);
+  font-weight: 800;
+  font-size: 40px;
+  padding-block: 1.4rem;
+}
+
+.link-container {
+  display: flex;
+  gap: 10px;
+}
+
+.link-container a {
+  text-decoration: none;
+  font-size: 20px;
+  letter-spacing: 2px;
+  color: var(--text-color);
+}
+
+.link-container a.active {
+  font-weight: 700;
+}
```

Comment:
- [ ] the link texts are not looking like what is on design, kindly fix the font color